src/System.Net.Requests/tests/HttpWebRequestTest.cs

```diff
@@ -603,52 +601,67 @@ public void Expect_Set100Continue_ThrowsArgumentException(Uri remoteServer)
         [Theory, MemberData(nameof(EchoServers))]
         public void DefaultMaximumResponseHeadersLength_SetAndGetLength_ValuesMatch(Uri remoteServer)
         {
-            int defaultMaximumResponseHeadersLength = HttpWebRequest.DefaultMaximumResponseHeadersLength;
-            const int NewDefaultMaximumResponseHeadersLength = 255;
-
-            try
-            {
-                HttpWebRequest.DefaultMaximumResponseHeadersLength = NewDefaultMaximumResponseHeadersLength;
-                Assert.Equal(NewDefaultMaximumResponseHeadersLength, HttpWebRequest.DefaultMaximumResponseHeadersLength);
-            }
-            finally
+            RemoteInvoke(() =>
```

Comment:
Why is this test a theory?  `remoteServer` never appears to be used. (Same goes for some of the other tests.)